src/platform-integration/linux/install-linux/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Add Linux devtools for Flutter
+description: Configure your system to develop Flutter for Linux.
+short-title: Add Linux to possible Flutter SDK targets
```

Comment:
That's not much shorter! Still long for a tab.

Reply:
True. I'll take a look at these. I wasn't happy with them either.